src/components/common/SafeHtmlWithImageLoader.jsx

```
@@ -0,0 +1,57 @@
+import { useEffect, useRef, useState } from 'react'
+import DOMPurify from 'dompurify'
+import Spinner from '../web/Spinner'
+
+const SafeHtmlWithImageLoader = ({ html }) => {
```

ESLint

[error] 'html' is missing in props validation

(react/prop-types)